packages/@n8n/task-runner/src/js-task-runner/__tests__/js-task-runner.test.ts

```diff
@@ -34,7 +34,7 @@ describe('JsTaskRunner', () => {
 				...defaultConfig.baseRunnerConfig,
 				grantToken: 'grantToken',
 				maxConcurrency: 1,
-				n8nUri: 'localhost',
+				taskBrokerUri: 'localhost',
```

Comment:
Shouldn't this have scheme and port?

https://github.com/n8n-io/task-runner-launcher/blob/37515673b39f9b61e996a98002543b0c638c019f/internal/config/validate_url_test.go#L29-L35

Reply:
It uses it directly [here](https://github.com/n8n-io/n8n/blob/rename-runner-n8n-uri-env-var/packages/@n8n/task-runner/src/task-runner.ts#L95). So it's not a URI. We should either rename it or make it use the uri